tests/candev/Makefile

```diff
@@ -7,7 +7,36 @@ USEMODULE += isrpipe
 # define the CAN driver you want to use here
 CAN_DRIVER ?= native
 
-ifeq ($(CAN_DRIVER), PERIPH_CAN)
+ifeq ($(CAN_DRIVER), MCP2515)
+    FEATURES_REQUIRED = periph_gpio periph_spi periph_gpio_irq
```

Comment:
As indicated earlier, this should go to the *drivers/Makefile.dep*

Reply:
Strangely enough, removing this line will break the entire test-app. Even if they are in drivers/Makefile.dep 
I added the line in the Makefile again for now, but this will probably need some more investigating, on why it didn't work with the modules in Makefile.dep.